config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml

```diff
@@ -47,3 +47,38 @@ periodics:
       resources:
         requests:
           memory: "8Gi"
+
+- name: ci-kubernetes-e2e-windows-gce
+  decorate: true
+  extra_refs:
+  - org: yujuhong
+    repo: gce-k8s-windows-testing
+    base_ref: master
+    path_alias: k8s.io/gce-k8s-windows-testing
+  interval: 2h
+  labels:
+    preset-k8s-ssh: "true"
+    preset-service-account: "true"
+  spec:
+    containers:
+    - command:
+      - runner.sh
+      - /workspace/scenarios/kubernetes_e2e.py
+      args:
+      - --check-leaked-resources
+      - --cluster=
+      - --extract=ci/latest
+      - --gcp-zone=us-west1-b
+      - --ginkgo-parallel=8
+      - --provider=gce
+      - --gcp-nodes=2
+      - --test=false
+      - --test-cmd=$GOPATH/src/k8s.io/gce-k8s-windows-testing/run-e2e.sh
+      - --test-cmd-args=--ginkgo.focus=\[Conformance\] --minStartupPods=8
+      - --timeout=720m
+      env:
+      - name: KUBE_GCE_ENABLE_IP_ALIASES
+        value: "true"
+      - name: NUM_WINDOWS_NODES
+        value: "3"
```

Comment:
I think `--gcp-nodes=2` can compute `NUM_WINDOWS_NODES=3`...

Reply:
IIUC, `--gcp-nodes` sets the `NUM_NODES`. After https://github.com/kubernetes/kubernetes/pull/73442, this sets only the # linux nodes.
`NUM_WINDOWS_NODES` is a new variable that configures the windows nodes, which will be used for testing.

Reply:
@pjh in case I'm wrong

Reply:
if you set one.. the other one will be set by kubetest like https://github.com/kubernetes/test-infra/blob/master/kubetest/main.go#L548-L605.. (We prefer flags, but k/k is still a bash fire so..)

Reply:
This looks right to me, and these two variables are the minimum required for a working Windows cluster.